projects/components/src/datagrid/datagrid.component.html

```diff
@@ -70,7 +71,7 @@ <h3 *ngIf="this.header" class="vcd-header">{{ header }}</h3>
     </clr-dg-column>
 
     <clr-dg-row
-        *ngFor="let restItem of items; let i = index"
+        *clrDgItems="let restItem of items; let i = index; trackBy: trackBy"
```

Comment:
I'm not sure we need this, we typically use `*ngFor` for server side data. I think this is so the grid can do filtering/sorting/paging.